tests/unit_test/chat/chat_test.py

```
@@ -1887,21 +1887,40 @@ def test_validate_media_file_type_invalid_type(tmp_path):
+@pytest.mark.asyncio
+@patch("kairon.shared.chat.user_media.UserMediaData.objects")
+def test_validate_media_file_type_file_already_exists(mock_objects, tmp_path):
+    test_file = tmp_path / "file.png"
+    test_file.write_bytes(b"x" * (1 * 1024 * 1024))
+    mock_upload = MagicMock()
+    mock_upload.filename = "file.png"
+    mock_upload.content_type = "image/png"
+    mock_upload.file = open(test_file, "rb")
+    mock_objects.return_value.first.return_value = MagicMock()
+
+    with pytest.raises(AppException, match="File file.png already exists"):
```

Ruff (0.14.2)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)